docs/index.md

```diff
@@ -1,226 +1,35 @@
 ---
 layout: index
-title: SW-Helpers
+title: Welcome to Workbox
 navigation_weight: 0
 ---
 
-<!-- DO NOT EDIT. This page is autogenerated. --><!-- To make changes, edit templates/README.hbs, not README.md! -->
-[![Travis Build Status][travis-image]][travis-url]
-[![AppVeyor Build status][appveyor-image]][appveyor-url]
-[![Dependency Status][dependency-image]][dependency-url]
-[![Dev Dependency Status][dev-dependency-image]][dev-dependency-url]
+# Welcome to Workbox
 
-# SW-Helpers
+Workbox is a suite of helper libraries for service workers and progressive web
+apps designed to simplify your development experience. 
 
-## Contents
+The easiest workbox deployments implement precaching in a manner of minutes.
+While the full set of features supports such things as runtime caching, routing,
+offline analytics, background syncing, and more.
 
-- [Ways to Use the Libraries](#ways-to-use-the-libraries)
-- [Setting Up](#setting-up)
-- [The Libraries](#the-libraries)
-  - [sw-appcache-behavior](#sw-appcache-behavior)
-  - [sw-broadcast-cache-update](#sw-broadcast-cache-update)
-  - [sw-build](#sw-build)
-  - [sw-cache-expiration](#sw-cache-expiration)
-  - [sw-cli](#sw-cli)
-  - [sw-lib](#sw-lib)
-  - [sw-offline-google-analytics](#sw-offline-google-analytics)
-  - [sw-precaching](#sw-precaching)
-  - [sw-routing](#sw-routing)
-  - [sw-runtime-caching](#sw-runtime-caching)
-- [External Contributions](#external-contributions)
-- [License](#license)
+## A service worker in two minutes
 
-## Ways to Use the Libraries
+Use [sw-cli](reference-docs/stable/latest/module-sw-cli.html#main)
+to add a service worker to an existing project in under two minutes.
 
-Despite the number of modules in this package, there are a number of approaches to using these tools which many be used separately or together in any number of combinations.
+## A build script in ten minutes
 
-* **Precaching, runtime caching and routing**: Use [sw-lib](#sw-lib) to quickly
-  implement a [caching strategies](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/) or a route.
-* **Build process**: Generate a service worker or manifest in your build process
-  using [sw-build](#sw-build).
-* **Command line**: Generate a service worker or manifest from the command line.
-* **Go beyond the basics**: Implement more advanced capabilities and more
-  sophisticated use cases using any of the other libraries, which are [described
-  below](#the-libraries).
+Generate a service worker in your build process
+using [sw-build](reference-docs/stable/latest/module-sw-build.html#main).
 
-## Setting Up
+## Much more
 
-Each module is installed separately using the command line [as listed below](#the-libraries). To use a particular module:
+Go beyond the basics. If you'd like to use the Workbox libraries, and you
+already have your own service worker, then
+[sw-lib](reference-docs/stable/latest/module-sw-lib.html#main) is a better option.
```

Comment:
"better option" than what? It sounds like we're telling the developer that `sw-lib` is a better option than their own service worker, and I don't think that's what you mean.

Can you reword along the lines of:

_...then using `sw-lib` is the ideal entry point._